.github/workflows/git_release_create.yml

```diff
@@ -0,0 +1,66 @@
+name: Create Release
```

Comment:
This contribution makes a lot of sense assuming we collapse on the semver versioning mechanism.